.vscode/settings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "editor.minimap.enabled": false,
+  "editor.wrappingIndent": "indent",
+  "editor.overviewRulerBorder": false,
+  "editor.lineHeight": 24,
+  "[markdown]": {
+    "editor.quickSuggestions": {
+        "comments": "on",
+        "strings": "on",
+        "other": "on"
+    }
+  },
+  "files.defaultLanguage": "markdown",
+  "files.exclude": {
+    "**/node_modules": true
+  },
+  "files.watcherExclude": {
+    "**/node_modules": true
+  },
+  "foam.edit.linkReferenceDefinitions": "withExtensions",
```

Comment:
@thanethomson, how can I make this setting global? This and another pair are relevant here.

I will get read of other configs from vscode.

Reply:
I think what you want is to configure your **User** settings. The `.vscode` folder here only configures **Workspace** settings. See https://code.visualstudio.com/docs/getstarted/settings

Reply:
Not really, I want this, which are essentially paths, only apply to this repository, not to override global/user configurations.

I don't know exactly what a Workspace means for vscode, so probably the best is to remove this dir, like you suggested.

Reply:
Oh, you can still have a `.vscode` directory in your own local version of the repo. Please just don't commit it (i.e. add it to a `.gitignore` file).